### PR TITLE
Add some metrics of threads and docs of worker CacheManager threads

### PR DESCRIPTION
**Pull request by flaming-archer**
### What changes are proposed in this pull request?
   Add some metrics  of master RPC, worker RPC/block serialized threads and add metrics docs of worker CacheManager threads

### Why are the changes needed?
Metrics for monitor system

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  no

### CODE REVIEW

**Comment by flaming-archer** [comment]
@maobaolong @dbw9580 @jiacheliu3 Hello , could you please help to review this pr, because you reviewed https://github.com/Alluxio/alluxio/pull/15101 before.

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -428,6 +428,18 @@ public MetricKey build() {
               + "Use this metric to monitor the RPC pressure on master.")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey MASTER_RPC_THREAD_ACTIVE_COUNT =
+      new Builder("Master.RpcThreadActiveCount")
+          .setDescription("Active count of the master rpc threads. "
+              + "Use this metric to monitor the RPC pressure on master.")
```
Comment (dbw9580) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:434]:
```suggestion
          .setDescription("The number of threads that are actively executing tasks in the master RPC executor thread pool. "
              + "Use this metric to monitor the RPC pressure on master.")
```

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 2]
```diff
@@ -428,6 +428,18 @@ public MetricKey build() {
...
+          .setMetricType(MetricType.GAUGE)
+          .build();
+  public static final MetricKey MASTER_RPC_THREAD_CURRENT_COUNT =
+      new Builder("Master.RpcThreadCurrentCount")
+          .setDescription("Current count of the master rpc threads. "
+              + "Use this metric to monitor the RPC pressure on master.")
```
Comment (dbw9580) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:440]:
```suggestion
          .setDescription("Current count of threads in the master RPC executor thread pool. "
              + "Use this metric to monitor the RPC pressure on master.")
```

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 3]
```diff
@@ -1911,6 +1950,18 @@ public MetricKey build() {
               + "Use this metric to monitor the RPC pressure on worker.")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey WORKER_RPC_THREAD_ACTIVE_COUNT =
+      new Builder("Worker.RpcThreadActiveCount")
+          .setDescription("Active count of the worker rpc threads. "
+              + "Use this metric to monitor the RPC pressure on worker.")
```
Comment (dbw9580) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:1956]: similar to above.

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 4]
```diff
@@ -1911,6 +1950,18 @@ public MetricKey build() {
...
+          .setMetricType(MetricType.GAUGE)
+          .build();
+  public static final MetricKey WORKER_RPC_THREAD_CURRENT_COUNT =
+      new Builder("Worker.RpcThreadCurrentCount")
+          .setDescription("Current count of the worker rpc threads. "
+              + "Use this metric to monitor the RPC pressure on worker.")
```
Comment (dbw9580) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java:1962]: also this

File: core/common/src/main/java/alluxio/metrics/MetricKey.java (hunk 1 above)
Comment (flaming-archer) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java]: updated

File: core/common/src/main/java/alluxio/metrics/MetricKey.java (hunk 2 above)
Comment (flaming-archer) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java]: updated

File: core/common/src/main/java/alluxio/metrics/MetricKey.java (hunk 4 above)
Comment (flaming-archer) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java]: updated

File: core/common/src/main/java/alluxio/metrics/MetricKey.java (hunk 3 above)
Comment (flaming-archer) [reply core/common/src/main/java/alluxio/metrics/MetricKey.java]: updated

**Comment by flaming-archer** [comment]
I think the fail test is not caused by my pr, because it was success before. So pls @maobaolong  @jiacheliu3 help me to review this pr.

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**